testing/matrix-sdk-test/src/test_json/keys_query_sets.rs

```diff
@@ -915,6 +915,24 @@ impl PreviouslyVerifiedTestData {
         ruma_response_from_json(&data)
     }
 
+    /// Device ID of Carol's signed device.
+    ///
+    /// The device is returned as part of
+    /// [`Self::carol_keys_query_response_signed`] and
+    /// [`Self::carol_keys_query_response_unsigned`].
+    pub fn carol_signed_device_id() -> &'static DeviceId {
+        device_id!("JBRBCHOFDZ")
+    }
+
+    /// Device ID of Carol's unsigned device.
+    ///
+    /// The device is returned as part of
+    /// [`Self::carol_keys_query_response_signed`] and
+    /// [`Self::carol_keys_query_response_unsigned`].
+    pub fn carol_unsigned_device_id() -> &'static DeviceId {
+        device_id!("BAZAPVEHGA")
+    }
+
```

Comment:
arguably this doesn't match the subject of the PR, but the main reason I'm adding them now is so that I can refer back to them in documentation that I'm going to add in the next commit...